jupyterhub/apihandlers/users.py

```diff
@@ -209,14 +218,16 @@ def post(self, name):
             raise web.HTTPError(400, "%s's server is not running" % name)
         self.set_server_cookie(user)
         # a service can also ask for a user cookie
-        # this code prevent to raise an error
+        # this code prevents to raise an error
+        # cause service doesn't have 'other_user_cookies'
         if getattr(current, 'other_user_cookies', None) is not None:
             current.other_user_cookies.add(name)
 
 
 default_handlers = [
     (r"/api/users", UserListAPIHandler),
     (r"/api/users/([^/]+)", UserAPIHandler),
-    (r"/api/users/([^/]+)/server", UserServerAPIHandler),
+    (r"/api/users/([^/]+)/servers", UserCreateServerAPIHandler),
```

Comment:
> I mean we should still allow the case where the user's URL is /user/:name/, not /user/:name/1/.

How do you think to handle this ?

Because in the case you start with /user/{name} (first server) and then you add a new server, what would be the name ? /user/{name}/1  or 2 ?

It has more sense to me /user/:name/:server_name (maybe more consistent) but probably I don't see a lot of compatibility problems :)